.github/workflows/build-image.yaml

```diff
@@ -27,7 +27,7 @@ jobs:
         uses: docker/metadata-action@v3
         with:
           images: |
-            ghcr.io/statevoicesnational/spoke
+            ghcr.io/moveonorg/moveon-spoke
```

Comment:
Navigating to ghcr.io/moveonorg/moveon-spoke I get a 404, but ghcr.io/moveonorg/spoke works – is that what it should be?

Reply:
[Pasting here for visibility to the open-source community]

The [ghcr.io/moveonorg/moveon-spoke](http://ghcr.io/moveonorg/moveon-spoke) URL should exist once the branch is merged into `main` since the workflow is run when a push is made on the `main` branch. I had to create a new [ghcr.io/moveonorg/moveon-spoke](http://ghcr.io/moveonorg/moveon-spoke) URL and could not longer push to the old [ghcr.io/moveonorg/spoke](http://ghcr.io/moveonorg/spoke) URL (403 forbidden error). I believe it's because this MoveOnOrg/Spoke is actually a new forked repo from the StateVoicesNational/Spoke repo, so lost the permissions to push to [ghcr.io/moveonorg/spoke](http://ghcr.io/moveonorg/spoke)